MohammadBallal/main.py

```diff
@@ -0,0 +1,117 @@
+from fastapi import FastAPI, HTTPException, Path
+from pydantic import BaseModel
+from uuid import UUID, uuid4
+from datetime import date
+from enum import Enum
+
+
+
+app = FastAPI()
+
+# Using enum for status and priority
```

Comment:
As i mentioned before, please write classes and models in another file

Reply:
I handled the matter based on the tutorial you presented, and you mentioned that in the coming days, we will deal with code organization and modules, BTW The required change is done.